src/components/_button.scss

```diff
@@ -180,6 +213,21 @@
   font-size: $fz-l;
 }
 
+@mixin sui-button-icon--small {
+  height: 16px;
+  width: 16px;
+}
+
+@mixin sui-button-icon--medium {
+  height: 20px;
+  width: 20px;
+}
+
+@mixin sui-button-icon--large {
+  height: 24px;
+  width: 24px;
+}
```

Comment:
What about defining `sui-icon--large` instead of `sui-button-icon--large`? we'll be able tu use it in other components.